README.md

```diff
@@ -46,6 +46,28 @@ Further options to add to the commands above:
 **Note on API support:** Validating lists of STAC items/collections (i.e. `GET /collections` and `GET /collections/:id/items`) is partially supported.
 It only checks the contained items/collections, but not the other parts of the response (e.g. `links`).
 
+### Config file
+
+You can also pass a config file via the `--config` option. Simply pass a file path as value. No other parameters can be set via CLI then.
```

Comment:
The normal way for this to work is that any parameters passed on the CLI *override* the configuration.

Reply:
I don't have the time to make this work right now unfortunately, the way it's implemented now was a low-hanging fruit though...

Reply:
Implemented (but still needs to be tested properly)

Reply:
You can now also override boolean values correctly.